tests/pipeline/test_runner.py

```diff
@@ -24,8 +24,31 @@
 
 def test_docker_runner():
```

Comment:
Can we add a separate test that checks the errors ?

https://github.com/ml6team/fondant/blob/main/tests/test_explorer.py#L164

Reply:
I've added two separate test cases for both situations, docker and docker compose is missing.